data/sea/10-mempool.h

```diff
@@ -66,66 +18,42 @@ static void imempool_add_block (imempool_t *pool)
         return ptr;                                                                                    \
     }
 
-static void * NOINLINE imempool_alloc_block (imempool_t *pool, size_t num_bytes)
+static void * NOINLINE imempool_alloc_block (anemone_mempool_t *pool, size_t num_bytes)
```

Comment:
I was thinking we might want to roll the alloc check into these functions. But maybe that's a bad idea